config/initializers/rack_attack.rb

```diff
@@ -1,31 +1,65 @@
 # frozen_string_literal: true
 
 class Rack::Attack
+  class Request
+    def authorization
```

Comment:
Is this duck typing to satisfy Doorkeeper::OAuth::Token.authenticate when we pass it `self`?

Reply:
Yes, it is. No other way to do it afaik.